docker-compose/pattern-1/docker-compose.yml

```diff
@@ -1,20 +1,100 @@
-version: '2'
+version: '2.1'
 services:
-  is_rdbms:
-    build:
-        context: ../
-        dockerfile: pattern-1/database/Dockerfile
+  mysql:
+    image: mysql:5.7.19
+    ports:
+      - 3306
     environment:
-        MYSQL_ROOT_PASSWORD: root
+        MYSQL_ROOT_PASSWORD: wso2carbon
+    volumes:
+      - ./mysql/scripts:/docker-entrypoint-initdb.d
+    healthcheck:
+      test: ["CMD", "mysqladmin" ,"ping", "-h", "localhost"]
+      interval: 10s
+      timeout: 60s
+      retries: 5
   identity-server:
-    build:
-      context: ../
-      dockerfile: pattern-1/identity-server/Dockerfile
+    image: wso2is:5.3.0
+    ports:
+      - 9443
+    healthcheck:
+        test: ["CMD", "curl", "-k", "-f", "https://localhost:9443/carbon/admin/login.jsp"]
+        interval: 10s
+        timeout: 120s
+        retries: 5
     environment:
-      - SLEEP=10
+      - SERVICE_PORTS=9443
+      - VIRTUAL_HOST=https://wso2is
+      - EXTRA_ROUTE_SETTINGS=ssl check verify none
+      - BALANCE=source
+    volumes:
+        - ./identity-server/carbon/repository/components/lib/mysql-connector-java-5.1.34-bin.jar:/home/wso2carbon/wso2is-5.3.0/repository/components/lib/mysql-connector-java-5.1.34-bin.jar
```

Comment:
Shall we simplify host mount path to something like ./identity-mount/... instead of ./identity-server/carbon/... ?

Reply:
Here we have created folders within the pattern-x folder for each service for managing their files:
```
pattern-1/mysql/
pattern-1/identity-server/
```
Therefore, I think its better to keep that convention for the first level of the folder structure. May be we can rename carbon to something like files. WDYT?

Reply:
Actually my concern was also .../carbon/... part, cannot we consider simply identity-server/ as the server home and go like ./identity-server/repository/... instead of having ./identity-server/carbon/repository/... ?

Reply:
Actually my concern was also .../carbon/... part, cannot we consider simply identity-server/ as the server home and go like ./identity-server/repository/... instead of having ./identity-server/carbon/repository/... ?